.github/workflows/pkgci_test_sharktank.yml

```diff
@@ -30,7 +30,12 @@ jobs:
             gpu: none
             runs-on: ubuntu-24.04
 
-          - name: rocm_hip
+          - name: rocm_hip_mi300
+            target: target_hip
+            gpu: gfx942
+            runs-on: linux-mi300-2gpu-ossci-iree-org
```

Comment:
Is this running the 2 GPU tests or do we also need to opt-in somehow? When logs are ready, can you link to them here showing which tests are running?

It's also fine to keep the scope of this PR small and just add a new 1 GPU test, or even fully separate the `test_sharktank.yml` changes from the `regression_test.yml` changes.

Reply:
I'm pretty sure it is running the 2 gpu test, but Ill link it when the logs finish

Reply:
https://github.com/iree-org/iree/commit/3a1fdbc33795a22ef735f86c5b46eca9848de166